src/utils/Utils.ts

```diff
@@ -103,7 +103,7 @@ export namespace Utils {
         if (!workspaceFolder) {
             if (workspace.workspaceFolders) {
                 const arrayOfPoms: string[][] = await Promise.all(workspace.workspaceFolders.map(getAllPomPaths));
-                return [].concat.apply([], ...arrayOfPoms);
+                return [].concat.apply([], arrayOfPoms);
```

Comment:
This line also fixes a potential issue existing for a long time:
Open a multi-root workspace folder, only one of them was indexed.... I'm surprised that the bug was introduced 2 months ago... and no one knew that.